apps/charterafrica/src/payload/globals/Footer.js

```diff
@@ -0,0 +1,227 @@
+import linkGroup from "../fields/linkGroup";
+
+const Footer = {
+  slug: "footer",
+  label: {
+    en: "Footer",
+    fr: "Bas de page",
+    pt: "Rodapé",
+  },
+  fields: [
+    {
+      name: "siteDescription",
+      label: {
+        en: "Site Description",
+        fr: "Description du site",
+        pt: "Descrição do Site",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    {
+      name: "projectDescription",
+      label: {
+        en: "Project Description",
+        pt: "Descrição do Projeto",
+        fr: "Description du projet",
+      },
+      type: "richText",
+      localized: true,
+      required: true,
+      admin: {
+        elements: ["h2", "h3", "h4", "h5", "h6", "ol", "ul", "link"],
+      },
+    },
+    linkGroup(),
+    {
+      name: "newsletter",
+      type: "group",
+      label: {
+        en: "Newsletter",
+        fr: "Newsletter",
+        pt: "Boletim de Notícias        ",
+      },
+      fields: [
+        {
+          name: "title",
+          required: true,
+          localized: true,
+          type: "text",
+          label: {
+            en: "Title",
+            fr: "Titre",
+            pt: "título",
+          },
+        },
+        {
+          name: "embedCode",
+          type: "code",
+          label: {
+            en: "Embed Code",
+            fr: "Code d'intégre",
+            pt: "Código Incorporado",
+          },
+          localized: true,
+          required: true,
+        },
+      ],
+    },
+    {
+      name: "logo",
+      required: true,
+      localized: true,
+      type: "group",
+      label: {
+        en: "Footer Logo",
+        pt: "Imagem",
+        fr: "Logo de pied de page",
+      },
+      fields: [
+        {
+          name: "title",
+          required: true,
+          localized: true,
+          type: "text",
+          label: {
+            en: "Footer Title",
+            fr: "Titre",
+            pt: "título",
+          },
+        },
+        {
+          name: "src",
```

Comment:
```suggestion
          name: "logo",
```

This can now be logo i.e. funder's logo.